### ISSUE TITLE
Sprint 3 – Scrum Meeting / Retrospective (18-12-2016) 

### ISSUE THREAD

**Issue by juzplayplay** [mewind/RSDSupermarketPOS#20]
Good morning everyone! Today is the last day of Sprint 3, and we have finished the payment module in the system. The Team Services shows that our overall build is successful. Good job guys!

**Comment by mewind** [comment]
Thats great! Thank you for everyone efforts that leads to the completion of the task.

**Comment by lkc442012** [comment]
Actually in developing the calculation of the grand total of the product selected by user, Sin Nee and I have facing some problems that the grand total does not display in the payment page.

**Comment by mewind** [comment]
Yaa, luckily we found the reason why we cannot build it successfully in the Team Services. We just forgot to include few line of the code in the build, and this cause the grand total cannot be displayed.

**Comment by huylin25** [comment]
So do you guys solve it already?

**Comment by lkc442012** [comment]
Yes, after we added that few line of codes, then everything is fine.

**Comment by juzplayplay** [comment]
Oh I see... How about Huy Lin? Do you counter any difficulties in this week?

**Comment by huylin25** [comment]
Overall, I do not facing any problems in the development of displaying payment details. Since Kar Cheng's work is quite clear and clean, Sin Nee and I just adding some codes inside the payment module to perform the display of the payment details.

**Comment by juzplayplay** [comment]
Okay, since there are not much problem in this week, I will conclude what we have done in this sprint: 
1) Calculate subtotal of the products everytime the user adding new product into cart
2) Different discount rate based on user's login (member, staff or non-member)
3) Cancel payment function
4) Adding grand total (including GST, discount and subtotal)
5) Update Payment (including delete, update product quantity)
6) Display payment details after each payment
7) Automatically logout after each payment

**Comment by mewind** [comment]
So the payment module is considered done now?

**Comment by juzplayplay** [comment]
Yes, thanks again for everyone efforts. If no other question, we can dismiss the meeting now.

**Comment by lkc442012** [comment]
Thank you Raymond for your summarization.

**Comment by juzplayplay** [comment]
You are welcome.
